uber/custom_tags.py

```diff
@@ -445,10 +445,12 @@ def render(self, context):
 class single_day_prices(template.Node):
     def render(self, context):
         prices = ''
+        skiprest = ''
```

Comment:
If I can recall what it looked like.

--
Robert Earl <r.a.earl@gmail.com>
FAX: 1-440-919-5086

On Mon, Oct 12, 2015 at 9:22 PM, Victoria Earl <notifications@github.com>
wrote:

> In uber/custom_tags.py
> <https://github.com/RAMSProject/rams/pull/3#discussion_r41818601>:
>
> > @@ -445,10 +445,12 @@ def render(self, context):
> >  class single_day_prices(template.Node):
> >      def render(self, context):
> >          prices = ''
> > +        skiprest = ''
>
> Can you please submit a PR for V2, then?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/RAMSProject/rams/pull/3/files#r41818601>.
>